app/addons/cors/__tests__/components.test.js

```diff
@@ -0,0 +1,215 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+import FauxtonAPI from "../../../core/api";
+import Views from "../components";
+import Actions from "../actions";
+import Resources from "../resources";
+import Stores from "../stores";
+import utils from "../../../../test/mocha/testUtils";
+import React from "react";
+import ReactDOM from "react-dom";
+import sinon from "sinon";
+import { shallow, mount } from 'enzyme';
+
+FauxtonAPI.router = new FauxtonAPI.Router([]);
+var assert = utils.assert;
```

Comment:
minor but could you make these `const`. Also you can do something like this:

```js
const {assert, restore} = utils;
```

Then you don't have to do `utils.restore` in the afterEach